### ISSUE TITLE
Insights: Need realtime updates in the method calls

### ISSUE THREAD

**Issue by yahu1031** [serverpod/serverpod#1369]
**Is your feature request related to a problem? Please describe.**
Yes. When ever I make a method call, I always need to refresh the screen.

**Describe the solution you'd like**
Need to be automatically listening the Method calls form the server.

**Comment by vlidholt** [comment]
Good idea. We will do this for a future update, but cannot give a timeline just now. :)
